packages/yaml_language_server/project.bri

```diff
@@ -0,0 +1,59 @@
+import * as std from "std";
+import { npmInstallGlobal } from "nodejs";
+import nushell from "nushell";
+
+export const project = {
+  name: "yaml_language_server",
+  version: "1.18.0",
+  packageName: "yaml-language-server",
+};
+
+export default function yamlLanguageServer(): std.Recipe<std.Directory> {
+  return npmInstallGlobal({
+    packageName: project.packageName,
+    version: project.version,
+  }).pipe((recipe) => std.withRunnableLink(recipe, "bin/yaml-language-server"));
+}
+
+export async function test(): Promise<std.Recipe<std.File>> {
```

Comment:
The test of this package was inspired from https://github.com/brioche-dev/brioche-packages/pull/637, since this package doesn't provide any argument to get its version.